resources/documentation/monitoring/ipmi_exporter.rst

```diff
@@ -0,0 +1,142 @@
+Ipmi_exporter
+=============
+
+The ipmi_exporter is an official Prometheus exporter, and can be found here:
+https://github.com/soundcloud/ipmi_exporter
+
+By default, the ipmi_exporter runs under the port 9290.
+
+To access the metrics, either do:
+
+.. code-block:: text
+
+  curl 'http://localhost:9290/ipmi?target=mgmt1-1-bmc&module=equipment_R_E4m'
+
+.. note::
+
+  here, the request is http://<ip address of where the exporter is located>/snmp?target=<name of the taget to get metrics from>&module=<module>
+
+.. note::
+
+  you can get the list of all modules here: */etc/ipmi_exporter/ipmi_config.yml*
+
+or you can access it directly in a browser. However, using curl can be handy,
+because you can grep the output, and do other nice things with it.
+
+You can get a list of all modules under */etc/ipmi_exporter/ipmi_config.yml*
+
+The ipmi_config file is automatically generated by the ansible prometheus_server
+role.
+
+You should get something like this:
+
+.. code-block:: text
+
+  # HELP ipmi_bmc_info Constant metric with value '1' providing details about the BMC.
+  # TYPE ipmi_bmc_info gauge
+  ipmi_bmc_info{firmware_revision="3.70",manufacturer_id="Super Micro Computer Inc. (10876)"} 1
+  # HELP ipmi_chassis_power_state Current power state (1=on, 0=off).
+  # TYPE ipmi_chassis_power_state gauge
+  ipmi_chassis_power_state 1
+  # HELP ipmi_fan_speed_rpm Fan speed in rotations per minute.
+  # TYPE ipmi_fan_speed_rpm gauge
+  ipmi_fan_speed_rpm{id="1679",name="P-FAN1"} 7000
+  ipmi_fan_speed_rpm{id="1746",name="P-FAN2"} 6600
+  ipmi_fan_speed_rpm{id="1813",name="S-FAN1"} 7300
+  ipmi_fan_speed_rpm{id="1880",name="S-FAN2"} 7500
+  ipmi_fan_speed_rpm{id="1947",name="MB-FAN4"} 7400
+  ipmi_fan_speed_rpm{id="2014",name="MB-FAN5"} 7100
+  # HELP ipmi_fan_speed_state Reported state of a fan speed sensor (0=nominal, 1=warning, 2=critical).
+  # TYPE ipmi_fan_speed_state gauge
+  ipmi_fan_speed_state{id="1679",name="P-FAN1"} 0
+
+Some of the information given are the following:
+
+* Fan speed
+* Power consumption in watts
+* Temperature (multiple sensors)
+* Voltage
+* Amperes
+* Etc…
+
+You can see more by looking at the local metrics.
+
+Alerts
+------
+
+All the alerts for the ipmi_exporter are stored under /etc/prometheus/alerts/ipmi.yml
+
+Some of them include :
+
+* High fan speed
+* Fan speed too high
+* High power consumption
+* Power consumption too high
+* Problem with powers supply
+* High ipmi temperature
+* Ipmi temperature too high
+* Scraping problem
+* High voltage
+* Voltage too high
+
+Start service
+-------------
+
+To start the service, simply type:
+
+.. code-block:: text
+
+  systemctl start ipmi_exporter
+
+You can change the BMC options of the inventory groups (compute nodes or
+management nodes) under */etc/ipmi_exporter/ipmi_config.yml*
+
+By default, it should look like this:
+
+.. code-block:: yaml
+
+  modules:
+
+  equipment_compute_SEQ2_CER:
+    user: super
```

Comment:
Please change this user/password.

Reply:
Why?

Reply:
sensitive information

Reply:
These are default credentials available in the documentation of the hardware platform. If this is considered sensitive, we might consider using other kind of equipments for the documentation.

Reply:
Solved.